File: pset6/mario/less/mario.py
import cs50

def main():
    h = get_height() + 1
    k = h - 2
    
    for i in range(1,h):
        for j in range(k):
            print(" ", end = "")
        
        for g in range(i):
            print("#", end = "")
        k -= 1
        print()

def get_height():
    while True:
        n = cs50.get_int("Height: ")
        
        if n > 0 and n < 9:
            break
    return n    
    
main()    

File: pset1/mario/mario.c
#include<stdio.h>
#include<cs50.h>

int get_height(void);

int main(void)
{
    int h = get_height();
    int k = h;
    string s = "#";
    string d = " ";
    
    int i, j, g; 
    for (i = 1; i <= h; i++)
    {
        for (g = 1; g < k; g++)
        {
            printf("%s", d);
        }
        for (j = 1; j <= i; j++)
        {
            printf("%s", s);
        }
        printf("\n");
        k--;
    }
}

int get_height(void)
{
    int h;
    do
    {
        h = get_int("%s", "Please type a number between 1 and 8: ");
    }
    while (h < 1 || h > 8);
    return h;
}